django/utils/feedgenerator.py

```diff
@@ -118,11 +118,13 @@ def __init__(self, title, link, description, language=None, author_email=None,
     def add_item(self, title, link, description, author_email=None,
             author_name=None, author_link=None, pubdate=None, comments=None,
             unique_id=None, unique_id_is_permalink=None, enclosure=None,
-            categories=(), item_copyright=None, ttl=None, updateddate=None, **kwargs):
+            enclosures=None, categories=(), item_copyright=None, ttl=None,
```

Comment:
To keep backwards compatibility, we need to add `enclosures=None` after `updateddate=None`.